src/main/java/persistence/sql/meta/TableInfo.java

```diff
@@ -0,0 +1,37 @@
+package persistence.sql.meta;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+import java.util.Objects;
+
+public class TableInfo {
+    private String tableName;
```

Comment:
```suggestion
public class TableInfo {
    private final String tableName;
```
불변을 지킨다면 어떤 장점을 가질 수 있을까요?